x-pack/plugins/secops/public/lib/keury/index.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { fromKueryExpression, toElasticsearchQuery } from '@kbn/es-query';
+import { StaticIndexPattern } from 'ui/index_patterns';
+
+export const convertKueryToElasticSearchQuery = (
+  kueryExpression: string,
+  indexPattern: StaticIndexPattern
+) => {
+  try {
+    return kueryExpression
+      ? JSON.stringify(toElasticsearchQuery(fromKueryExpression(kueryExpression), indexPattern))
+      : '';
+  } catch (err) {
+    return '';
+  }
+};
+
+export const escapeQueryValue = (val: string | undefined) =>
+  val && val.replace ? val.replace(/"/g, '\\"') : '';
```

Comment:
I thought I commented on this earlier but maybe my comment didn't take or something weird so I apologize if you already marked this one as reviewed.

When would a string value not have a the function replace on it?

Couldn't this just be:

```ts
export const escapeQueryValue = (val: string | undefined) =>
  val ? val.replace(/"/g, '\\"') : '';
```

Or a one liner where you're using a default value setter if string is undefined like so:

```ts
export const escapeQueryValue = (val: string = '') => val.replace(/"/g, '\\"');
```